### ISSUE TITLE
EssentialsX breaks Narrate Chat in Accessibility Options

### ISSUE THREAD

**Issue by korttwulfe** [EssentialsX/Essentials#5619]
### Type of bug

Compatibility issue

### `/ess dump all` output

https://essentialsx.net/dump.html?id=3073ae358fbe4efd9a89609b018f578b

### Bug description

When EssentialsX plugin is installed on Spigot Server the Narrate All and Narrate System in Accessibility options (Java edition) works fine.  When Narrate Chat is selected no chat output is heard.

EssentialsX-2.20.1 &
EssentialsX-2.21.0-dev+25-fbfd7e9
spigot-1.20.4

### Steps to reproduce

1. Narrate Chat enabled in Single or Multiplayer works fine
2. Install EssentialsX plugin for Spigot Server.
3. Narrate Chat stops working in Single and Multiplayer.
4. Remove EssentialsX plugin.
5. Narrate Chat starts working again.

### Expected behaviour

When Narrate Chat is enabled in Accessibility Options the items typed by players as "chat" should be spoken via text to speech.

### Actual behaviour

When Narrate Chat is selected in Accessibility options the player chat is not spoken with text to speech.

**Comment by pop4959** [comment]
Narrate chat works just fine with the latest dev build of Essentials and EssentialsChat installed.
